packages/edit-site/src/components/resizable-frame/index.js

```diff
@@ -148,7 +164,7 @@ function ResizableFrame( { isFull, children } ) {
 				bottomLeft: false,
 				topLeft: false,
 			} }
-			resizeRatio={ isOversized ? 1 : 2 }
+			resizeRatio={ 2 }
```

Comment:
From what we gathered yesterday, changing the `resizeRatio` dynamically was the main culprit for the bug we were experiencing. It looks like `ResizableBox` (and, more specifically, the third-party library that it uses internally) don't support this usecase correctly.

For now, we've decided to keep the ratio fixed to 2, as it seemed the best compromise for now.

Reply:
The issue with keeping it constant is mouse position becomes desynchronized when it crosses the threshold.